modules/common/src/main/java/com/opengamma/sdk/common/ServiceInvoker.java

```diff
@@ -157,6 +158,33 @@ public Response intercept(Chain chain) throws IOException {
     }
   }
 
+  //An interceptor that adds the User-Agent header and exposes useful information about the SDK and runtime.
+  private class UserAgentHeaderInterceptor implements Interceptor {
+
+    @Override
+    public Response intercept(Chain chain) throws IOException {
+      Properties systemProperties = System.getProperties();
```

Comment:
Always need to check for `SecurityException` when accessing properties - its one of those things a security manager often restricts.